compiler/src/org.graalvm.compiler.truffle.compiler/src/org/graalvm/compiler/truffle/compiler/nodes/frame/NewFrameNode.java

```diff
@@ -268,12 +269,13 @@ public void virtualize(VirtualizerTool tool) {
             }
         }
 
-        tool.createVirtualObject(virtualFrameObjectArray, objectArrayEntryState, Collections.<MonitorIdNode> emptyList(), false);
+        NodeSourcePosition sourcePosition = getNodeSourcePosition();
```

Comment:
This seems like as good an answer as we can provide.

Reply:
Ok, thanks for confirming.